BUILD.md

```diff
@@ -10,13 +10,14 @@ git clone https://github.com/dm-vdo/kvdo
 
 Build the uds library:
 ~~~
-make -C $BUILD_DIR/vdo/utils/uds/
+cd   $BUILD_DIR/kvdo
+make -C /usr/src/kernels/`uname -r` M=`pwd`
```

Comment:
It is necessary to build the user space UDS library. It is totally unnecessary to build the kvdo kernel module. The only reason kvdo is needed is to provide block.c, which for some reason is not in the user space library.